### ISSUE TITLE
make it easier to contribute

### ISSUE THREAD

**Issue by oren** [cayleygraph/site-content#1]
It would be nice to add to the readme the exact commands for installing and using hugo. I know it can be found on hugo's website but doing that will make it easy to contribute by people who are not familiar with hugo.

**Comment by robertmeta** [comment]
Yeah, I had hoped this would be "simplier" but I fear I made a mistake.  It might be easier to just have all the stuff for hugo inlined here, which is what @barakmich wanted to do from the start anyway.  Expect an update later today.
